docs/src/components/landingBlocks/HeroBlock.tsx

```diff
@@ -1,14 +1,21 @@
-import React from "react";
-import Section from "../Section";
-import CTA from "../ui/CTA";
+import React from 'react';
+import Section from '../Section';
+import CTA from '../ui/CTA';
+import { styled } from 'linaria/react';
+import { appTheme } from '../../themes';
+import Images from '../../assets/images';
+import useWindowSize from '../../hooks/useWindowSize';
+
+const { media, breakpoints } = appTheme;
+const { HeroPetsImage, HeroGraphicsImage } = Images;
 
 const content = {
```

Comment:
Please consider here and in other blocks:

```suggestion
const defaultContent = {
```

Reply:
done

Reply:
done